UnityProject/Assets/Scripts/UI/IngameMenu/SetVolume.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+public class SetVolume : MonoBehaviour
+{
+		
```

Comment:
It might be wise to set the actual UI Sliders value on enable. Incase we want to save the volume preference in the future and load it on application start